packages/generator-volto/README.md

```diff
@@ -38,18 +38,19 @@ Usage:
   yo @plone/volto:app [<projectName>] [options]
 
 Options:
-  -h,   --help           # Print the generator's options and usage
-        --skip-cache     # Do not remember prompt answers                                        Default: false
-        --skip-install   # Do not automatically install dependencies                             Default: false
-        --force-install  # Fail on install dependencies error                                    Default: false
-        --ask-answered   # Show prompts for already configured options                           Default: false
-        --volto          # Desired Volto version, if not provided, the most recent will be used
-        --canary         # Desired Volto version should be a canary (alpha)                      Default: true
-        --interactive    # Enable/disable interactive prompt                                     Default: true
-        --skip-addons    # Don't ask for addons as part of the scaffolding
-        --addon          # Addon loader string, like: some-volto-addon:loadExtra,loadOtherExtra
-        --workspace      # Yarn workspace, like: src/addons/some-volto-addon
-        --description    # Project description
+  -h,   --help              # Print the generator's options and usage
+        --skip-cache        # Do not remember prompt answers                                        Default: false
+        --skip-install      # Do not automatically install dependencies                             Default: false
+        --force-install     # Fail on install dependencies error                                    Default: false
+        --ask-answered      # Show prompts for already configured options                           Default: false
+        --volto             # Desired Volto version, if not provided, the most recent will be used
+        --canary            # Desired Volto version should be a canary (alpha)                      Default: false
+        --interactive       # Enable/disable interactive prompt                                     Default: true
+        --skip-addons       # Don't ask for addons as part of the scaffolding
+        --addon             # Addon loader string, like: some-volto-addon:loadExtra,loadOtherExtra
+        --workspace         # Yarn workspace, like: src/addons/some-volto-addon
+        --description       # Project description
+        --defaultAddonName  # The name of the add-on project added to the project by default
```

Comment:
I'm not sure what this option does. Does this option pertain to the add-on that the developer will create for the generated project? Assuming so:

```suggestion
        --defaultAddonName  # The add-on's default name to be added to the generated project.
```

Reply:
I'd say it's better this:

`The default add-on's name to be added to the generated project.`

added in the last commit.

Reply:
@stevepiercy I think it's fine as it is, the suggested text is no longer true.

Reply:
I'm sorry, I'm not clear about what _is_ true for this option.